core/services/feeds/orm.go

```diff
@@ -349,3 +372,24 @@ FROM job_proposals
 
 	return count, nil
 }
+
+// IsJobManaged determines if a job is managed by the feeds manager.
+func (o *orm) IsJobManaged(ctx context.Context, jobID int64) (bool, error) {
+	stmt := `
+SELECT exists (
+	SELECT 1
+	FROM job_proposals
+	INNER JOIN jobs ON job_proposals.external_job_id = jobs.external_job_id
+	WHERE jobs.id = ?
+);
+`
+
+	var exists bool
+	if err := o.db.WithContext(ctx).Raw(stmt, jobID).Scan(&exists).Error; err != nil {
+		if err != sql.ErrNoRows {
```

Comment:
This should never happen, EXISTS will always return exactly one row

Reply:
Fixed